### ISSUE TITLE
SUGGESTION: registration/login required after a (low) number of edits from the same IP

### ISSUE THREAD

**Issue by JOM13** [segler-alex/radiobrowser#99]
Probably this kind of measure can prevent or at least decrease vandalism. Don't know if it's too cumbersome.

**Comment by NiceMusic** [comment]
Forget it ... will use a lot of VPN and rotation email adresses too
Have a Nice Day

**Comment by gkaklas** [comment]
I agree. Either that or require an account to make any edit at all (the attacker could use a proxy to change their IP address). I understand that the point was probably that anyone could add and edit radio stations, but if you think about it, every community-sourced dataset requires an account as well (Wikipedia, OpenStreetMap, OpenFoodFacts etc).

Is the vandalism automated? If so, a captcha could work too. :thinking: 

Blocking edits is really counter-productive as there are ~100 broken radio stations right now that we can't fix for the last few months.

**Comment by rgctoronto** [comment]
I started helping with the site about 18 months ago and there have always been 100+ "broken" links.   That's not really new.

However, maybe an interim solution until a better solution is found might be to give a small number of "trusted" users edit permissions if it's possible to create granular permissions.

**Comment by jazzyeagle** [comment]
I know it's been awhile since anyone posted on this, but I wanted to add my support for this option.  By requiring users to register/login in order to make changes, if anyone vandalizes the data, that user's changes can be undone relatively easily and their account suspended from making changes while allowing others to continue making changes.

**Comment by Riotous** [comment]
@segler-alex Please add login support. Every good wiki has it anyway.

**Comment by alexwender1** [comment]
@gkaklas and @rgctoronto we have to show our interest in helping with the project.

Some form of credential is required to access the database. And delegate roles to anyone who is willing to help. Who are these users?

**Comment by jazzyeagle** [comment]
@alexwender1: Currently, you allow anyone to add (previously edit) without credentials at all.  What I was thinking/recommending was simply allowing anyone to sign up to add/edit, however if anyone *does* spam or do something malicious, that user's credentials would be disabled.  That way, it can still be a community effort to maintain the database, but with a little bit of administrative control to prevent spamming to the best of your ability.

**Comment by alexwender1** [comment]
> @alexwender1: Currently, you allow anyone to add (previously edit) without credentials at all. What I was thinking/recommending was simply allowing anyone to sign up to add/edit, however if anyone _does_ spam or do something malicious, that user's credentials would be disabled. That way, it can still be a community effort to maintain the database, but with a little bit of administrative control to prevent spamming to the best of your ability.

I agree with you. There must be some form of accreditation...

Even as a form of protection... it is good to create groups of reviewers (by country or language). If there is a problem, it will be easier to identify the person responsible.

**Comment by Riotous** [comment]
They should really give a small group of people unique logins to make edits and take their credentials away if they vandalize anything.
